depends/packages/rapidcheck.mk

```diff
@@ -0,0 +1,21 @@
+package=rapidcheck
+
+$(package)_version:1.0
+
+$(package)_download_path:https://github.com/Christewart/rapidcheck/releases/download/1.0
```

Comment:
Again, I think a subtree is the better choice per my previous comment. Also, I don't think we can have someone verify this binary each time it is bumped, even if it was done deterministically.